docs/manual/resource-leak-checker.tex

```diff
@@ -539,6 +541,13 @@
 not possible, the programmer will probably need to suppress a warning and
 then verify the code using a method other than the Resource Leak Checker.
 
+
+\sectionAndLabel{Resource Leak Checker Annotation Inference Algorithm}{resource-leak-checker-inference-algo}
+
+There is a special inference mechanism to infer annotations for the Resource Leak Checker. By default, this mechanism operates when the -Ainfer flag is enabled. To utilize the whole-program inference mechanism outlined in section \ref{whole-program-inference}, you should provide the \<-AenableWPIForRLC> flag.
+
+For more detail, please read \href={https://arxiv.org/pdf/2306.11953.pdf}{Automatic Inference of Resource Leak Specifications}.
```

Comment:
I would phrase this differently: as it is, it's not clear to a reader that this is a reference to a research paper. How about the following:
```
The paper \href={https://arxiv.org/pdf/2306.11953.pdf}{Automatic Inference of Resource Leak Specifications}, published at OOPSLA 2023, gives more details on the inference algorithm.
```

Reply:
Updated to the one you suggested.